.circleci/config.yml

```diff
@@ -188,7 +92,7 @@ workflows:
               only: /^\d+\.\d+\.\d+$/
             branches:
               ignore: /.*/
-      - deploy_svn_tag:
+      - wp-svn/deploy:
```

Comment:
Replaces the whole job with the [wp-svn orb](https://circleci.com/developer/orbs/orb/studiopress/wp-svn#usage-deploy) job